protocol/consistencychecks.go

```diff
@@ -2,6 +2,7 @@
 // on data received from a CONIKS directory.
 // These include data binding proof verification,
 // and non-equivocation checks.
+// TODO: move all STR-verifying functionality to a separate module
```

Comment:
Could we open an issue for that so we won't forget about it? Or is this planned to happen in this PR?